src/GraphQL.Federation/Instrumentation/FederatedInstrumentFieldMiddleware.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GraphQL.Instrumentation;
+using GraphQL.Utilities;
+
+namespace GraphQL.Federation.Instrumentation
+{
+    /// <summary>
+    /// Middleware required for Apollo federated tracing to record performance metrics of field.
+    /// </summary>
+    public class FederatedInstrumentFieldMiddleware : IFieldMiddleware
+    {
+        /// <inheritdoc/>
+        public async Task<object> Resolve(IResolveFieldContext context, FieldMiddlewareDelegate next)
+        {
+            var metadata = new Dictionary<string, object>
+            {
+                { "responseName", context.FieldDefinition.Name},
+                { "type", SchemaPrinter.ResolveName(context.FieldDefinition.ResolvedType) },
```

Comment:
`SchemaPrinter.ResolveName` method was deleted.

Reply:
Thank you! Removed